.tekton/operator-next-bundle-pull-request.yaml

```diff
@@ -36,9 +36,6 @@ spec:
     value: 5d
   - name: dockerfile
     value: .konflux/olm-catalog/bundle/Dockerfile
-  - name: build-platforms
-    value:
-      - linux/x86_64
```

Comment:
If the `build-platforms` parameter is omitted, is there a default? My assumption based on this change is it defaults to `[linux/x86_64, linux/aarch64, linux/ppc64le, s390x]`, but wanted to confirm